### PR TITLE
feat: create reusable hooks for staking and rewards

### PR DESCRIPTION
**Pull request by miliPaoletti**

### CODE REVIEW

File: apps/mission/src/components/mission/dappStore/staking/Card.tsx [hunk 1]
```diff
@@ -7,17 +7,15 @@ import { BalanceContainer } from "../card/BalanceContainer";
 import { Card } from "../card/Card";
 import { Description } from "../card/Description";
 import { Title } from "../card/Title";
-import { useStakingInfo } from "../../../../internal/functionality/hooks/useStakingInfo";
-import useAssetsTopBar from "../../../../internal/functionality/hooks/useAssetsTopBar";
 import { useSelector } from "react-redux";
-import { StoreType } from "evmos-wallet";
+import { StoreType, useAssets, useStake } from "evmos-wallet";
```
Comment (sandoche) [inline apps/mission/src/components/mission/dappStore/staking/Card.tsx:11]: can we mege line 11 and 18?

File: packages/evmos-wallet/src/hooks/rewards/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/apps/blob/main/LICENSE)
+
+import { BigNumber } from "ethers";
+import { convertFromAtto } from "helpers";
+import { WalletExtension } from "../../internal/wallet/functionality/wallet";
+export const getBalance = (amount: BigNumber, wallet: WalletExtension) => {
```
Comment (sandoche) [inline packages/evmos-wallet/src/hooks/rewards/helpers.ts:7]: Are all these helpers tested?

**Review by sandoche — APPROVED** [review]
All good except the two small comments

File: packages/evmos-wallet/src/hooks/rewards/helpers.ts (hunk 2 above)
Comment (miliPaoletti) [reply packages/evmos-wallet/src/hooks/rewards/helpers.ts:7]: no, I'll add the tests :)

**Review by sandoche — APPROVED** [review]
Looks good

**Pull request merged**